synapseclient/client.py

```diff
@@ -1354,6 +1355,115 @@ def uploadFileHandle(self, path, parent, synapseStore=True, mimetype=None, md5=N
         """
         return upload_file_handle(self, parent, path, synapseStore, md5, file_size, mimetype)
 
+    ############################################################
+    #                  Download List                           #
+    ############################################################
```

Comment:
I think it's a good idea to keep the naming of these functions consistent with the end point.